__tests__/index.test.js

```diff
@@ -218,4 +218,64 @@ describe('/api/articles/:article_id/comments', () => {
         expect(response.body.msg).toBe('Bad request');
       });
   });
+})
+
+describe('/api/articles/:article_id', () => {
+  test('PATCH:201 inserts a new comment to the comments table and sends the new comment back to the client', () => {
+    const patchObj = {
+      inc_votes : 100
+    };
+    return request(app)
+      .patch('/api/articles/3')
+      .send(patchObj)
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article.length).toBe(1);
+        const article = response.body.article[0];
+        expect(article).toMatchObject({
+          title : expect.any(String),
+          topic : expect.any(String),
+          author : expect.any(String),
+          body : expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String)
+        })
+      });
```

Comment:
This simply asserts that we get back an article but not that it has actually be changed in anyway. We should be expecting the votes to have increased here so using an exact value would be the way to prove this instead of any number which doesnt show anything has changed.